docs/appendix/changelog.md

```diff
@@ -19,6 +19,13 @@ outputs:
 
 <!-- If there is no concrete date for a change that makes sense, use the end of the month it was released in. -->
 
+{{% changelog date="2024-05-09" color="changed" title="Return type of GetImage()" %}}
+
+The Python SDK introduced a new image container class called [`ViamImage`](https://python.viam.dev/autoapi/viam/components/camera/index.html#viam.components.camera.ViamImage).
+The camera API's [`GetImage()`](/components/camera/#getimage) API method now returns a `ViamImage` type, and the vision service's [GetDetections()](/ml/vision/#getdetections) and [GetClassifications()](/ml/vision/#getclassifications) take in `ViamImage` as a parameter.
+
```

Comment:
do we need to provide a code sample for how to use this now? I think there's a helper method to convert a viamimage to a pilimage, we should add that here otherwise people won't know - info on this is on DOCS-2218

Reply:
Added code sample!